pkg/controllers/provisioning/scheduling/scheduler.go

```diff
@@ -51,11 +52,18 @@ type Scheduler struct {
 }
 
 type Schedule struct {
-	*v1alpha5.Constraints
+	*RuntimeConstraints
 	// Pods is a set of pods that may schedule to the node; used for binpacking.
 	Pods []*v1.Pod
 }
 
+type RuntimeConstraints struct {
```

Comment:
Thoughts on calling this object scheduling.Constraints for parity with v1alpha5.Constraints. I don't think that "Runtime" differentiates more meaningfully than the local context.

Reply:
Also -- I'm not understanding why this object is useful.